### DISCUSSION TITLE
AD & CP Harvester unload and destination issues

### DISCUSSION THREAD

**Discussion by TangoMike1982** [Courseplay/Courseplay_FS22#2348] [General]
Recently starting to experience issues with AD 2.0.0.9 in combination with CP 7.2.0.2 when unloading harvesters to transport the crops to a certain destination to fulfill contracts. Lets say the required destination for the harvested crop is "Feed and Grain South", after unloading the harvester when the tractor starts to leave the field and switches from CP to AD it randomly selects a new destination i.e. "Red Marble Bowling" or any other available point of sale, instead of driving to the preselected destination.

**Comment by Tensuko** [comment]
That is not a CP Issue.
We do not interact with the AD Hud and destinations.

**Comment by rohne83** [comment]
This is AD feature. I guess you clicked on the destination item in AD hud, so that this item (not the select target point, but the symbol) appears in blue instead of white. This means that AD will search itself for the unloading station with best selling price, and drive there.
Just click on the symbol again, so that it gets white, and AD will drive to the originally selected unloading station.
